process/txsimulator/txSimulator.go

```diff
@@ -202,22 +220,24 @@ func (ts *transactionSimulator) adaptSmartContractResult(scr *smartContractResul
 	}
 
 	if scr.OriginalSender != nil {
-		resScr.OriginalSender = ts.addressPubKeyConverter.Encode(scr.OriginalSender)
+		resScr.OriginalSender = ts.addressPubKeyConverter.SilentEncode(scr.OriginalSender, log)
 	}
 	if scr.RelayerAddr != nil {
-		resScr.RelayerAddr = ts.addressPubKeyConverter.Encode(scr.RelayerAddr)
+		resScr.RelayerAddr = ts.addressPubKeyConverter.SilentEncode(scr.RelayerAddr, log)
```

Comment:
use Encode or SilentEncode everywhere in this file. However, wrap the errors before returning them